string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,33 @@
+/* exported capitalizeWord */
+
+// storage for output
+// if the word all lowercased is equal to 'javascript':
+//   -for each character of the String, starting at the beginning:
+//     -if the character lowercased is equal to 'j' or 's':
+//       -add the character uppercased to the output storage
+//     -otherwise:
+//       -add the character lowercased to the output storage
+// otherwise:
+//   add the first character uppercased to the output storage
+//   for each character of the string, starting at the second character:
+//     -add the character lowercased to the output storage
+// return the output storage
+
+function capitalizeWord(word) {
+  var newWord = '';
+  if (word.toLowerCase() === 'javascript') {
+    for (var i = 0; i < word.length; i++) {
+      if (word[i].toLowerCase() === 'j' || word[i].toLowerCase() === 's') {
+        newWord += word[i].toUpperCase();
+      } else {
+        newWord += word[i].toLowerCase();
+      }
+    }
```

Comment:
In a special case like this, you already know the value you want to return, so there's no need to make the computer go through the steps of creating it, you can just `return "JavaScript"`